README.md

```diff
@@ -49,6 +49,12 @@ Select a cargo command from the submenu. This spawns a shell with [Terminus](htt
 
 ![Example](./images/commands.gif)
 
+### LSP-rust-analyzer: Join Lines
+
+Joins lines accounting for rust-specific logic.
+
+Also bound to the default join-lines key binding.
```

Comment:
```suggestion
Also bound to the default join-lines key binding `primary+shift+j`. 
```

Reply:
maybe?

Reply:
I don't know if people understand "primary" but could list mac and linux/windows ones separately.

Reply:
Sure, my intention was to make it more obvious what the keybinding is.